content/04.study.md

```diff
@@ -47,6 +47,24 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Inference, Imputation, and Prediction
+
+Deep learning approaches are beginning to help address some of the current limitations of feature-by-feature analysis approaches to DNA methylation data, and may help uncover additional important features necessary to understand the biological underpinnings behind different pathological states.
+One of the more popular applications is the prediction of the degree of methylation at CpG sites neighboring measured sites.
+DeepSignal employs a convolutional neural network to construct features from raw electrical Nanopore signals from sites near a methylated base, and concatenates uses a bi-directional recurrent neural network on DNA sequences of the aligned signals to detect methylation [@tag:Ni2018].
+DeepCpG applies a similar method using scBS-Seq, DNA sequence and Bidirectional GRUs [@tag:Angermueller2017] and methods like DAPL and DeepMethyl incorporate sequence and topological structure [@tag:Qiu2018] [@tag:Khwaja2017] [@tag:Wang2016_methyl] [@tag:Fu2019].
+In addition to this, Gene expression has been used to infer and impute methylation states [@tag:Peng2019] [@tag:Levy-Jurgenson2018], methylation of genes predicted from promoter methylation [@tag:Pan2018], and convolutional models have been able to predict methylation status from images [@tag:Momeni2018][@tag:Korfiatis2017].
+While these examples of methylation imputation and inference methods have value it is imperative to recognize limitations of imputing cytosine modifications.
+Not only does imputing DNA methylation share similar limitations with  genotype imputation, but there is additional complexity in correlation of DNA methylation that can depend on cell type and other variables related with DNA methylation that can vary by sample.
+As the abundance of available tissue types and cell types with whole-genome bisulfite sequencing (and oxidative bisulfite sequencing), grows, the accuracy of imputation methods for DNA methylation is expected to increase.
```

Comment:
```suggestion
As the number of tissue types and cell types with whole-genome bisulfite sequencing (and oxidative bisulfite sequencing) grows, the accuracy of DNA methylation imputation is expected to increase.
```

Reply:
I made this change to emphasize that it seems what you're saying is that many of the current limitations are available data in addition to methods.